compiler/ghc/default.nix

```diff
@@ -20,7 +20,7 @@ let self =
 , # GHC can be built with system libffi or a bundled one.
   libffi ? null
 
-, useLLVM ? !stdenv.targetPlatform.isx86
+, useLLVM ? !stdenv.targetPlatform.isx86 && !stdenv.targetPlatform.isGhcjs
```

Comment:
I'm still not sure, I think we now lost 9.2+ aarch64 NCG.

Reply:
We are going to fix this in another PR